src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -299,8 +333,91 @@ function BoundaryDropdown({
       setSelectedBoundaries={newSelectedBoundaries => {
         dispatch(setSelectedBoundariesRedux(newSelectedBoundaries));
       }}
+      labelMessage={labelMessage}
     />
   );
 }
 
+export const SelectMenu = withStyles(() => {
+  return {
+    root: {
+      '& label': {
+        textTransform: 'uppercase',
+        letterSpacing: '3px',
+        fontSize: '11px',
+      },
+      '& svg': { color: 'white', fontSize: '1.25rem' },
+    },
+  };
+})(SimpleBoundaryDropdown);
+
+export const BoundaryDropdownMap = () => {
+  const map = useSelector(mapSelector);
+  const selectedBoundaries = useSelector(getSelectedBoundaries);
+  const dispatch = useDispatch();
+
+  const boundaryLayerData = useSelector(layerDataSelector(boundaryLayer.id)) as
+    | LayerData<BoundaryLayerProps>
+    | undefined;
+  const { data } = boundaryLayerData || {};
+
+  const styles = useStyles();
+
+  if (!data || !map) {
+    return null;
+  }
+
+  return (
+    <div className={styles.dropdownMenu}>
+      <CenterFocusWeak fontSize="small" className={styles.icon} />
+      <SelectMenu
+        selectedBoundaries={selectedBoundaries}
+        labelMessage="Go to"
+        className={styles.formControl}
+        setSelectedBoundaries={newSelectedBoundaries => {
+          dispatch(setSelectedBoundariesRedux(newSelectedBoundaries));
```

Comment:
Is this the same redux used for selecting boundaries that I made? I think it should be fine to use it, just want to make sure it won't break anything if you're using both drop-downs simultaneously/cause highlights if you're using the alert window.

Reply:
I created an internal state here d5d3c84a2aecd2a652255af256f06d124848e901